farm/modeling/language_model.py

```diff
@@ -1354,3 +1388,203 @@ def load(cls, pretrained_model_name_or_path, language=None, **kwargs):
             camembert.model = CamembertModel.from_pretrained(str(pretrained_model_name_or_path), **kwargs)
             camembert.language = cls._get_or_infer_language_from_name(language, pretrained_model_name_or_path)
         return camembert
+
+
+class DPRQuestionEncoder(LanguageModel):
+    """
+    A DPRQuestionEncoder model that wraps HuggingFace's implementation
+    """
+
+    def __init__(self):
+        super(DPRQuestionEncoder, self).__init__()
+        self.model = None
+        self.name = "dpr_question_encoder"
+
+    @classmethod
+    def from_scratch(cls, vocab_size, name="dpr_question_encoder", language="en"):
```

Comment:
Is this always giving back a freshly initialized bert base model? Please add a short doc string

Reply:
no, it gives back a DPR model with DPR weights. I'll add option to load weights from bert-base-uncased

Reply:
Removed from_scratch() method from DPRQuestionEncoder and DPRContextEncoder